access_point/server/server.py

```diff
@@ -17,20 +18,30 @@ def install():
 
 @app.route('/config/<stt>/<tts>/<hotword>/<wake>')
 def config(stt, tts, hotword, wake):
-    subprocess.call(['sudo', 'bash' , server_folder + '/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
-    return 'Done' # pylint-enable
+    subprocess.call(['sudo', 'bash' , access_point_folder + '/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
+    display_message = {"Configuration":"Successful"}
+    resp = jsonify(display_message)
+    resp.status_code = 200
+    return resp # pylint-enable
 
 @app.route('/auth/<auth>/<email>/<passwd>')
 def login(auth, email, passwd):
-    subprocess.call(['sudo', 'bash', server_folder + '/login.sh', auth, email, passwd]) #nosec #pylint-disable type: ignore
-    return 'Authenticated' # pylint-enable
+    subprocess.call(['sudo', 'bash', access_point_folder + '/login.sh', auth, email, passwd]) #nosec #pylint-disable type: ignore
+    display_message = {"Authentication":"Successful"}
+    resp = jsonify(display_message)
+    resp.status_code = 200
+    return resp # pylint-enable
 
 @app.route('/wifi_credentials/<wifissid>/<wifipassd>')
 def wifi_config(wifissid,wifipassd):
     wifi_ssid = wifissid
     wifi_password = wifipassd
-    subprocess.call(['sudo', 'bash', access_point_folder + '/wifi_search.sh', wifi_ssid, wifi_password])  #nosec #pylint-disable type: ignore
-    return 'Wifi Configured' # pylint-enable
+    subprocess.call(['sudo', 'bash', wifi_search_folder + '/wifi_search.sh', wifi_ssid, wifi_password])  #nosec #pylint-disable type: ignore
+    display_message = {"Wifi":"Configured"}
```

Comment:
Looks good to me.
Have you tested it locally?

Reply:
Yes @prateekiiest  , its working now